server/server.go

```diff
@@ -131,6 +131,7 @@ func Start() {
 
 	router := httprouter.New()
 	router.GET("/", indexHandler)
+	router.GET("/healthz", healthHandler)
```

Comment:
Why `healthz` and not `health`?

Reply:
It's a historic way of defining _health_ endpoint and to prevent collision and visually identify them that they are not actual endpoints to be used by consumers. Example in official [k8s doc](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request).

Reply:
Thanks for the explanation :+1: